src/engraving/rw/read206/read206.cpp

```diff
@@ -3028,9 +3028,9 @@ static void readBox(Box* b, XmlReader& e, ReadContext& ctx)
     b->setBottomMargin(0.0);
```

Comment:
Aren't these first 4 lines redundant? They only set properties to values that are already their defaults. And these are `NOSTYLE` properties, so no style flags need to be set either, for these properties.

Reply:
You're right, they are redundant.

Reply:
And the problem I tried to solve with those property flags stemmed from scores I first migrated from Mu1 to Mu2, so it is now solved there only